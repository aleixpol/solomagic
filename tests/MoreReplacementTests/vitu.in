\ref something0
\mt somethingsomething vitovito somethingsomething

\ref something1
\mt somethingsomething tivona somethingsomething

