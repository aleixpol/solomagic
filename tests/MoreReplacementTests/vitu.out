\ref something0
\mt somethingsomething vitovito somethingsomething
\ma somethingsomething vituvitu somethingsomething

\ref something1
\mt somethingsomething tivona somethingsomething
\ma somethingsomething tivona somethingsomething
